packages/js/src/Modules/Verto/webrtc/helpers.ts

```diff
@@ -563,14 +593,59 @@ function getWebRTCSupportedBrowserList(): Array<IWebRTCSupportedBrowser> {
           features: ['video', 'audio'],
           supported: SUPPORTED_WEBRTC.full,
         },
-        { browserName: 'Firefox', features: ['audio'], supported: SUPPORTED_WEBRTC.partial },
+        {
+          browserName: 'Firefox',
+          features: ['audio'],
+          supported: SUPPORTED_WEBRTC.partial,
+        },
         { browserName: 'Safari', supported: SUPPORTED_WEBRTC.not_supported },
-        { browserName: 'Edge', features: ['audio'], supported: SUPPORTED_WEBRTC.partial },
+        {
+          browserName: 'Edge',
+          features: ['audio'],
+          supported: SUPPORTED_WEBRTC.partial,
+        },
       ],
     },
   ];
 }
 
+function createAudio(file, id): HTMLAudioElement | null {
+  if (file && id) {
+    const ringAudio = document.createElement('audio');
+    ringAudio.id = id;
```

Comment:
We should to check if an element with this ID exists first, in case the client is initialized multiple times.